chart/cas-cif/values-dev.yaml

```diff
@@ -38,3 +38,6 @@ deploy-db:
 
 download-dags:
   airflowEndpoint: https://cas-airflow-dev.apps.silver.devops.gov.bc.ca
+
+ggircs:
+  namespace: 9212c9-dev
```

Comment:
this doesn't need to be hardcoded: it can be set in the makefile, using something like:
```
install: GGIRCS_NAMESPACE=$(CIF_NAMESPACE_PREFIX)-$(ENVIRONMENT)
```
and setting ggircs.namespace in the `HELM_OPTS` variable